database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -602,7 +602,7 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
     runTransaction(action)
   }
 
-  protected def addMetadata(events: Iterable[Metadatum])
+  override def addMetadata(events: Iterable[Metadatum])
```

Comment:
Following the tech talk, does `Metadatum` counts as a simple type or a business logic type?